plugins/schema/graphql-kotlin-federated-hooks-provider/README.md

```diff
@@ -0,0 +1,4 @@
+# GraphQL Kotlin Federated Hooks Provider
+
+This module provides default implementation of `SchemaGeneratorHooksProvider` that can be used to generate federated GraphQL
```

Comment:
Do we need to call it `federated-hooks-provider`? How about just `hooks-provider`. Choosing regular or federated hooks is just the internal logic and you might not get federated hooks